### PR TITLE
[locale][esMX] Update translations

### PR DESCRIPTION
**Pull request by bernardnoguez**
Solved [locale][koKR] conflicts, reviewed and added esMX translations

### CODE REVIEW

File: Localization/Translations/DebugMessages.lua [hunk 1]
```diff
@@ -55,7 +55,7 @@ local debugMessagesLocales = {
         ["ruRU"] = "состояние трекера Questie. Добавьте 'show', 'hide', 'reset', чтобы показать, скрыть или сбросить трекер.",
         ["deDE"] = "Zeigen/Verstecken des Questie Trackers. Bei 'show', 'hide' oder 'reset' wird der Tracker explizit gezeigt, versteckt oder zurückgesetzt",
         ["koKR"] = nil,
-        ["esMX"] = nil,
+        ["esMX"] = "Muestra el Rastreador. Agrega 'mostrar', ''ocultar y 'reestablecer' para mostrar/ocultar y reestablecer el Rastreador",
```
Comment (BreakBB) [inline Localization/Translations/DebugMessages.lua:58]: There are two `'` which doesn't seem right.

**Review by BreakBB — CHANGES_REQUESTED** [review]
Thanks @bernardnoguez for the fast change! 👍🏻 You could also have reused the other PR and simply push more commits onto the existing ones 😀 

I think there is a small issue, but I might be wrong 😅

File: Localization/Translations/DebugMessages.lua (hunk 1 above)
Comment (bernardnoguez) [reply Localization/Translations/DebugMessages.lua:58]: You're absolutely right, fixing it. I kinda messed up the merge, that's why I created a new PR, but I might take the offer to soak some more knowledge on how to be more proficient at this.

**Review by BreakBB — APPROVED** [review]
Looks fine now, thanks @bernardnoguez 👍🏻 

Sure, go ahead and hit me up on Discord or simply ask in another PR 😄

**Pull request merged**